scene/gui/label.cpp

```diff
@@ -36,6 +36,34 @@
 
 #include "servers/text_server.h"
 
+Ref<Font> Label::_get_font() const {
+	switch (header_mode) {
+		case HEADER_SMALL:
+			return get_theme_font("font_header_small");
+		case HEADER_MEDIUM:
+			return get_theme_font("font_header_medium");
+		case HEADER_LARGE:
+			return get_theme_font("font_header_large");
+		default:
+			return get_theme_font("font");
+	}
+	return Ref<Font>();
+}
+int Label::_get_font_size() const {
+	switch (header_mode) {
+		case HEADER_SMALL:
+			return get_theme_font_size("font_header_small_size");
+		case HEADER_MEDIUM:
+			return get_theme_font_size("font_header_medium_size");
+		case HEADER_LARGE:
+			return get_theme_font_size("font_header_large_size");
+		default:
+			return get_theme_font_size("font_size");
+	}
+
+	return 1;
```

Comment:
This is not necessary if you handle the "default" case. Same above.

Reply:
Hmm.. Thanks, I was getting compiler warnings before, but I added the default later on, let me see if its still showing a warning.